### DISCUSSION TITLE
Cite JupyterLite

### DISCUSSION THREAD

**Discussion by Erik4444** [jupyterlite/jupyterlite#890] [General]
Hello,
I'm currently writing my bachelor thesis about digitalization of worksheets in school and i want to mention JupyterLite.
Is it possible to cite your work and if, then how?

Thanks in advance!

**Comment by bollwyvl** [comment]
No DOIs of yet, have added to the agenda for next week's workshop.

In the meantime, you could perhaps cite some of the proceedings of conferences where talks on JupyterLite have been given:
- https://pydata.org/global2021/schedule/presentation/113/jupyterlite-jupyter-webassembly-python/
- https://pydata.org/london2022/events/jupyterlite-jupyter-webassembly-python/
- https://scipy-2022-jupyterlite-virtual-poster.readthedocs.io/en/latest/README.html

**Reply by Erik4444** [reply]
Okay, thanks a lot!

**Comment by jtpio** [comment]
Thanks @Erik4444 for the interest.

You might also want to link to blog posts like the following ones:

- https://blog.jupyter.org/jupyterlite-jupyter-%EF%B8%8F-webassembly-%EF%B8%8F-python-f6e2e41ab3fa
- https://blog.jupyter.org/jupyter-everywhere-f8151c2cc6e8
- https://blog.jupyter.org/mamba-meets-jupyterlite-88ef49ac4dc8

**Comment by pierrepo** [comment]
- A DOI could be obtained fairly easily with a code archiving in Zenodo. 
- But maybe a short article in https://joss.theoj.org/ (or even https://jose.theoj.org/ if you want to stress on the benefit for education) could be a better option?
